crates/typst-pdf/src/gradient.rs

```diff
@@ -155,22 +161,39 @@ fn shading_function(ctx: &mut PdfContext, gradient: &Gradient) -> Ref {
 
         // If the color space is HSL or HSV, and we cross the 0°/360° boundary,
         // we need to create two separate stops.
-        if gradient.space() == ColorSpace::Hsl || gradient.space() == ColorSpace::Hsv {
+        let hue_component_index = match gradient.space() {
```

Comment:
I think it would be preferable to have a `hue_index` method on `ColorSpace` that returns `Option<usize>` and use that instead. That way it's nice and localized.